src/makieplotting.jl

```diff
@@ -16,7 +16,7 @@ end
 Solutionplot produces the classical contour plot onto the finite element mesh. Most important
 keyword arguments are:
 
-- `field::Symbol=:default` representing the field which gets plotted, defaults to the first field in the `dh`.
+- `field::Symbol=:u` representing the field which gets plotted, defaults to the first field in the `dh`.
```

Comment:
Can we default to nothing and use the first symbol in the first field handler as default?

Reply:
Docstring.